demo/llm.rag.service/chat-serveragllmpluslb.yaml

```diff
@@ -13,10 +13,12 @@ spec:
     metadata:
       labels:
         model: serveragllm
+      annotations:
+        node.elotl.co/instance-type-regexp: "^t3.xlarge$"
```

Comment:
Thanks for adding this to work around the issue that pickle/unpickle the faiss representation btw intel/amd
x86_64 doesn't work.  As we know, this does restrict our service to working on AWS, which is fine for now.

Once we move away from using pickle/unpickle to store/retrieve the data, this won't be needed, but it does
bring up the issue that Luna does not support a cross-cloud way to express a preference for intel or amd family; we could open a low-priority luna feature ticket to add this if we end up seeing customer interest.

Reply:
Good point. Thanks, Anne. I will track this as a Luna feature ticket.

CC @githubjbw - So he can be in the loop if we see customer interest.

Reply:
Agreed, it would be a nice feature but probably is a low value/low interest feature. I'll log a ticket for tracking so we don't loose track. As mentioned, the best solution is to remove any dependencies that would require this for our own POCs/examples.

Reply:
Thank you for offering to create a ticket and tracking this @githubjbw. 👍